### DISCUSSION TITLE
Any plans to add date & time to the search?

### DISCUSSION THREAD

**Discussion by dalicode** [adirh3/Fluent-Search#875] [Ideas]
![image](https://github.com/adirh3/Fluent-Search/assets/50684889/91025bc9-e572-4051-88d7-330f4619dfa8)

Since using fluent search I have my taskbar mostly hidden but as a result that also hides my date & time. Having date & time in fluent search would be useful. It also looks good imo.

**Comment by adirh3** [comment]
This is already implemented in next nightly release :) 
(probably this/next week)
